build-system/tasks/extension-helpers.js

```diff
@@ -384,8 +386,11 @@ async function doBuildExtension(extensions, extension, options) {
  * @param {?Object} options
  */
 function watchExtension(extDir, name, version, latestVersion, hasCss, options) {
-  const watchFunc = function () {
-    removeFromBabelCache(extDir);
+  const watchFunc = function (modifiedFile) {
```

Comment:
nit: While you're here, shall we standardize all three `watchFunc`s in this PR to use arrow notation?

Reply:
I actually think these functions look better as old-style funcs :)